tests/system/src/testapp7/java/mymod/myapp7/DataUrlWithModuleLayer.java

```diff
@@ -29,38 +29,44 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 import javafx.application.Application;
-import javafx.application.Platform;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.concurrent.Worker.State;
 import javafx.scene.Scene;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.web.WebView;
 import javafx.stage.Stage;
 
 public class DataUrlWithModuleLayer extends Application {
+    public static final int ERROR_OK = 0;
+    public static final int ERROR_ASSUMPTION_VIOLATED = 1;
```

Comment:
We usually leave 1 unused, since that is what `Process.waitFor` returns if the application fails to launch (in case that happens it will be easier for someone to diagnose).